src/renderer/services/appUpdate/service.ts

```diff
@@ -0,0 +1,20 @@
+import * as FP from 'fp-ts/function'
+import * as O from 'fp-ts/Option'
+import * as Rx from 'rxjs'
+import * as RxOp from 'rxjs/operators'
+
+import { observableState } from '../../helpers/stateHelper'
+
+type AppUpdater = O.Option<string>
+
+const { get$: appUpdater$, set: setAppUpdater } = observableState<AppUpdater>(O.none)
+
+FP.pipe(
+  Rx.from(window.apiAppUpdate),
+  RxOp.catchError(() => Rx.EMPTY),
```

Comment:
Handle error (we do need to know if something went wrong)

Reply:
ok. in which way ? show something to the user or just internal `console.log` ?

Reply:
Show same modal "New version is available", but with an error message in it's body: "`Check for update failed: ${error.?message ?? error.toString()`" or similar